### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Pazzo97**
Below are the tasks we did in this In this pull request, for the Capstone project

- Set up webpack for the project
- Set up linters configuration for lint errors
- Set up Jest for testing
- Display the episodes list dynamically on the homepage from the API
- Add the CSS for the project
- Display the total number of episodes on the homepage
- Display the comment popup where users can add comments
- Display total comments of an episode
- Display details of an episode
- Display likes number
- Add Test function for total comments and total episodes
- Added a video presentation of the project

### CODE REVIEW

File: src/modules/add-comment.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+async function addComment(itemid, usrName, usrComment) {
```
Comment (Lameck1) [inline src/modules/add-comment.js]: - When we look at the learning objectives of this capstone project, we find this objective 👇🏼:

  > - Use ES6 syntax.
  
  Therefore, we are also looking for strict adherence to ES6 syntax when implementing our functions in this review.
  
  Please refactor all functions that are not in ES6 syntax to be ES6 compliant. We'll need to remember about `arrow functions` here:
  
  ```JavaScript
  const functionName = (params_if_any) => {
    // function body here
  }
  ```
  
  If you would like to read more about arrow functions, check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) out.

File: src/modules/add-likes.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+async function addLikes(itemid) {
```
Comment (Lameck1) [inline src/modules/add-likes.js]: - [ ] Same case as above, please refactor accordingly.

File: src/modules/episode-counter.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+async function episodeCounter(url) {
```
Comment (Lameck1) [inline src/modules/episode-counter.js]: - [ ] Same case as above, please refactor accordingly.

File: src/modules/fetch-data.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+async function fetchData(url) {
```
Comment (Lameck1) [inline src/modules/fetch-data.js]: - [ ] Same case as above, please refactor accordingly.

File: src/modules/load-comment.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  } else {
+    const div = document.createElement('div');
+    const template = '<div>--- Be the first to comment ---</div>';
+
+    div.innerHTML = template;
+    list.appendChild(div);
+  }
+};
+
+async function loadComment(itemid) {
```
Comment (Lameck1) [inline src/modules/load-comment.js]: - [ ] Same case as above, please refactor accordingly.

File: src/modules/load-likes.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+async function loadLikes() {
```
Comment (Lameck1) [inline src/modules/load-likes.js]: - [ ] Same case as above, please refactor accordingly.

File: src/modules/total-comment.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+async function totalComment(itemid) {
```
Comment (Lameck1) [inline src/modules/total-comment.js]: - [ ] Same case as above, please refactor accordingly.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @Pazzo97 & @rolandfbk, 

Good job so far! Well done for giving it you all 👏🏼. You've one awesome project implementation here 💪🏼.
There are just a few issues for us to iron out before we could move ahead.

### Some highlights for you
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Nice video presentation ✔️ 
* In the project:
   * Clean codebase ✔️ 
   * API interaction properly implemented using `async/await`✔️  
   * All expected features of the MVP functioning as expected ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Pazzo97 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**